django_extensions/management/debug_cursor.py

```diff
@@ -8,13 +8,15 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends import utils
 
+from django_extensions.settings import DEFAULT_PRINT_SQL_TRUNCATE_CHARS
+
 
 @contextmanager
 def monkey_patch_cursordebugwrapper(print_sql=None, print_sql_location=False, truncate=None, logger=print, confprefix="DJANGO_EXTENSIONS"):
     if not print_sql:
         yield
     else:
-        truncate = getattr(settings, '%s_PRINT_SQL_TRUNCATE' % confprefix, 1000)
+        truncate = truncate or getattr(settings, '%s_PRINT_SQL_TRUNCATE' % confprefix, DEFAULT_PRINT_SQL_TRUNCATE_CHARS)
```

Comment:
I personally like something like the following better:

```python
if truncate is None:
    truncate = getattr(settings, '%s_PRINT_SQL_TRUNCATE' % confprefix, DEFAULT_PRINT_SQL_TRUNCATE_CHARS)
```

or

```python
truncate = truncate if truncate else getattr(settings, '%s_PRINT_SQL_TRUNCATE' % confprefix, DEFAULT_PRINT_SQL_TRUNCATE_CHARS)
```

but that is nit picking for sure ;-)